likecoin/js/admin/likecoin_metabox.js

```diff
@@ -76,15 +128,57 @@ function onSubmitToISCN(e) {
   window.addEventListener('message', onISCNCallback, false);
 }
 
+async function onEstimateAndUploadArweave(e) {
+  e.preventDefault();
+  try {
+    const res = await jQuery.ajax({
+      type: 'POST',
+      url: `${wpApiSettings.root}likecoin/v1/posts/${wpApiSettings.postId}/arweave/estimate`,
+      method: 'POST',
+      beforeSend: (xhr) => {
+        xhr.setRequestHeader('X-WP-Nonce', wpApiSettings.nonce);
+      },
+    });
+    const { ipfsHash, LIKE, memo } = res;
+    if (LIKE === '0' || LIKE === 0) {
+      // exact same files are already exist in Arweave.
+      // skip Arweave upload flow to directly register an ISCN
+      const { arweaveId } = res;
+      const arweaveInfo = {
+        ipfsHash,
+        arweaveId,
+      };
+      submitToISCNAfterArweaveUpload(arweaveInfo);
```

Comment:
I think should no directly submit to ISCN? Because we do have an another ISCN button in this metabox. We should just update the arweave id into database, then update the UI allowing user to click upload to ISCN button.